lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -142,6 +142,78 @@ testRule({
 			endLine: 1,
 			endColumn: 138,
 		},
+		{
+			code: `a {
```

Comment:
[suggest] I recommend using `stripIndent` like other test codes:

```js
import { stripIndent } from 'common-tags';
```

```suggestion
			code: stripIndent`a {
```

If doing so, we won't need to care about `column` values.

Reply:
It didn't change `endColumn` and `column`.

Reply:
It didn't change `endColumn` and `column`.
What's the desired outcome? I don't get it.

Reply:
Sorry, I mean the following should be:

```js
			code: stripIndent`
				a {
					border-top-width: 1px;
					border-right-width: 2px;
					border-bottom-width: 3px;
					border-left-width: 4px;
					border-top-color: red;
					border-right-color: green;
					border-bottom-color: blue;
					border-left-color: yellow;
					border-top-style: dotted;
					border-right-style: solid;
					border-bottom-style: dashed;
					border-left-style: double;
				}
			`,
			fixed: stripIndent`
				a {
					border-width: 1px 2px 3px 4px;
					border-color: red green blue yellow;
					border-style: dotted solid dashed double;
				}
			`,
```